GitVersionCore.Tests/Mocks/MockCommitLog.cs

```diff
@@ -8,7 +9,10 @@ public class MockCommitLog : ICommitLog, ICollection<Commit>
 
     public IEnumerator<Commit> GetEnumerator()
     {
-        return Commits.GetEnumerator();
+        if (SortedBy == CommitSortStrategies.Reverse)
```

Comment:
Yeah, so I think this is right.

`Commits.Add()` will add to the bottom of the list internally. So to replicate the commit log we call `.Reverse()`, but if you pass that flag I don't reverse.

I think this is all good